libpod/define/errors.go

```diff
@@ -34,6 +34,10 @@ var (
 	// aliases.
 	ErrNoAliases = errors.New("no aliases for container")
 
+	// ErrMissingPlugin indicates that the requested operation requires a
+	// plugin that is not present on the system or in the configuration.
+	ErrMissingPlugin = errors.New("required plugin missing")
```

Comment:
```suggestion
	ErrMissingVolumePlugin = errors.New("required volume plugin missing")
```

To leave space for other types of plugins in the future.

Reply:
I think having one is probably fine - "missing plugin" on a Volume command is pretty distinct from "missing plugin" in a network command